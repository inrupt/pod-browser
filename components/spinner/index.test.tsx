import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Spinner  from './index'

describe('Spinner', () => {
    test('Renders a spinner', () => {
        const tree = shallow(<Spinner />);
        expect(shallowToJson(tree)).toMatchSnapshot();
    });
});