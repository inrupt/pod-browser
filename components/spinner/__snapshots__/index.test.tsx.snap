// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Renders a spinner 1`] = `
<WithStyles(ForwardRef(Container))>
  <WithStyles(ForwardRef(CircularProgress))
    className="makeStyles-MuiCicularProgress-root-1"
    color="primary"
  />
</WithStyles(ForwardRef(Container))>
`;
